python/aitemplate/compiler/transform/fuse_split.py

```diff
@@ -250,6 +251,11 @@ def _fuse_split_and_strided_op(sorted_graph: List[Tensor]) -> List[Tensor]:
                 and len(output.dst_ops()) == 1
                 for next_op in output.dst_ops()
             )
+            can_optimize_split_cat &= len(output.dst_ops()) > 0 and all(
```

Comment:
This new flag would make us miss some fusion opportunities, e.g.:

```
x1, x2, x3 = split(t)
y1 = concat(x1, x2)
y2 = gemm(x3, t2)
```

Because both concat and gemm are fusible, I think we could be able to remove the split in this case. However, neither ```can_optimize_split_cat``` nor ```can_fuse_split``` is True. 

Perhaps we should stick with a single flag ```can_fuse_split``` and make it cover the concat case as well.